azure_arc_servers_jumpstart/aws/scale_deployment/ansible/terraform/ansible_config/group_vars/all.yml

```diff
@@ -0,0 +1,7 @@
+---
+    # Variables here are applicable to all host groups
+    
```

Comment:
I would give the option to also set up proxy if the environment needs it: 

    # proxy: <Uncomment if connectivity is restricted by a proxy, format http://<proxyURL>:<proxyport> >